client/components/editor/Toolbar/index.vue

```diff
@@ -63,9 +63,11 @@ export default {
 
 <style lang="scss" scoped>
 .toolbar-wrapper {
-  position: fixed;
   width: 100%;
-  z-index: 99;
+
+  ::v-deep .element-toolbar-wrapper {
```

Comment:
Since the element-toolbar-wrapper is just a toolbar resolver and this one is an element for placing it within the editor, consider keeping the positioning within this one.